### PR TITLE
Feat/multi currency checkout

### PR DESCRIPTION
**Pull request by schwepps**

### CODE REVIEW

File: app/components/Payment/Quote.tsx [hunk 1]
```diff
@@ -1,100 +1,181 @@
...
 import { GreenButton } from "../Buttons/ActionButton";
+import { NotificationSource } from "~/utils/notifications/sources";
+import { TransactionStatus, num } from "starknet";
+import { useAccount, useContractWrite } from "@starknet-react/core";
+import { useNotifications } from "~/root";
+import { getStarkscanUrl } from "~/utils/utils";
+import { type BuildSwapTransaction, type Quote, type QuoteRequest, fetchBuildExecuteTransaction, fetchQuotes } from "@avnu/avnu-sdk";
+import _ from "lodash";
+
+const USDC_CONTRACT = "0x053c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8";
```
Comment (MartianGreed) [inline app/components/Payment/Quote.tsx:11]: mainnet address

File: app/components/Portfolio/Markeplaces.tsx [hunk 2]
```diff
@@ -3,15 +3,15 @@ import { useNotifications } from "~/root";
...
-    const { defautlNetwork } = useNotifications();
+    const { defaultNetwork } = useNotifications();
 
     return (
         <div className="absolute invisible top-0 left-0 bg-transparent group-hover:bg-dark-40 group-hover:visible w-full h-[100%] rounded-[8.8%]">
             <div className="relative w-full h-100%">
-                <a href={`${defautlNetwork === 'testnet' ? ASPECT_TESTNET_LINK : ASPECT_LINK}/asset/${project.address}/${num.hexToDecimalString(project.tokens[0].token_id)}`} rel="noreferrer" target="_blank" className="absolute top-6 right-16 md:top-4 w-10 h-10 rounded-full p-2 flex items-center justify-center bg-black/20 backdrop-blur-md cursor-pointer border border-neutral-300 hover:bg-black/5 hover:backdrop-blur-lg">
+                <a href={`${defaultNetwork === 'testnet' ? ASPECT_TESTNET_LINK : ASPECT_LINK}/asset/${project.address}/${num.hexToDecimalString(project.tokens[0].token_id)}`} rel="noreferrer" target="_blank" className="absolute top-6 right-16 md:top-4 w-10 h-10 rounded-full p-2 flex items-center justify-center bg-black/20 backdrop-blur-md cursor-pointer border border-neutral-300 hover:bg-black/5 hover:backdrop-blur-lg">
```
Comment (MartianGreed) [inline app/components/Portfolio/Markeplaces.tsx:11]: did this var points to "testnet" that changed to goerli in networks.ts

File: app/components/Project/Selection/ProjectInformation.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import { useProject } from "../ProjectWrapper";
+import ProjectStatusWithMilestone from "../Common/ProjectStatusWithMilestone";
+import ProjectKPIs from "./ProjectKPI";
+import BoostAndPoints from "./BoostAndPoints";
+import SaleProgress from "./SaleProgress";
+
+export default function ProjectInformation() {
+    const { project } = useProject();
+
+    // TODO: Replace with real KPIs
```
Comment (MartianGreed) [inline app/components/Project/Selection/ProjectInformation.tsx:11]: is it still todo ?

**Review by MartianGreed — DISMISSED** [review]

File: app/components/Payment/Quote.tsx (hunk 1 above)
Comment (schwepps) [reply app/components/Payment/Quote.tsx:11]: It's a fake component. We can leave it here.

File: app/components/Project/Selection/ProjectInformation.tsx (hunk 3 above)
Comment (schwepps) [reply app/components/Project/Selection/ProjectInformation.tsx:11]: No. I'll remove the comment.

File: app/components/Portfolio/Markeplaces.tsx (hunk 2 above)
Comment (schwepps) [reply app/components/Portfolio/Markeplaces.tsx]: This component is not used anymore. I just deleted it.

**Pull request merged**